Source/Core/DolphinWX/NetWindow.cpp

```diff
@@ -476,7 +519,7 @@ NetPlayDiag::NetPlayDiag(wxWindow* const parent, const CGameListCtrl* const game
 
 	wxPanel* const panel = new wxPanel(this);
 
-	// top crap
```

Comment:
Removing the word "crap" is one thing, but the comment looks a bit useless to me at all. Maybe find a more useful description; or remove it altogether?